.github/workflows/Build.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions-rust-lang/setup-rust-toolchain@v1
-      - run: cargo test
+      - run: ./ripple test
```

Comment:
As we discussed, I don't think this is best practice  - the unit tests should run with only `cargo test`, and not require a wrapper to setup the environment

Reply:
I agree, will remove these changes.